lib/commons/dom/create-grid.js

```diff
@@ -221,20 +239,8 @@ function isStackingContext(vNode, parentVNode) {
   }
 
   // a flex item or gird item with a z-index value other than "auto", that is the parent element display: flex|inline-flex|grid|inline-grid,
-  if (zIndex !== 'auto' && parentVNode) {
-    const parentDsiplay = parentVNode.getComputedStylePropertyValue('display');
-    if (
-      [
-        'flex',
-        'inline-flex',
-        'inline flex',
-        'grid',
-        'inline-grid',
-        'inline grid'
-      ].includes(parentDsiplay)
-    ) {
-      return true;
-    }
+  if (zIndex !== 'auto' && parentVNode && isFlexOrGridContainer(parentVNode)) {
```

Comment:
Might as well have `isFlexOrGridContainer` handle `null`. The function makes plenty sense if it returns false when you pass it null.

```suggestion
  if (zIndex !== 'auto' && isFlexOrGridContainer(parentVNode)) {
```